src/components/DataSession.vue

```diff
@@ -45,18 +50,17 @@ onMounted(() => {
 
 </script>
 
-
 <template>
   <v-container class="d-lg-flex">
     <v-row v-if="images.length">
       <v-col
-        v-for="image in images"
-        :key="image.basename"
+        v-for="image of images"
+        :key="image[0].basename"
```

Comment:
Is there a reason we need to do this indexing of 0? Is images an array of objects, so each object just has a basename and a url? Or am I missing something